erpnext/setup/doctype/currency_exchange_settings/currency_exchange_settings.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021, Wahni Green Technologies Pvt. Ltd. and contributors
```

Comment:
This doctype/settings belong to accounting. It was there in the setup folder mostly because of legacy code. There's no reason to keep it that way now 😅